rc.conf

```diff
@@ -11,6 +11,10 @@
 #HARDWARECLOCK="UTC"
 
 # Set timezone, availables timezones at /usr/share/zoneinfo.
+#
+# NOTE: it's preferred to set the timezone in /etc/localtime instead:
```

Comment:
`NOTE: This is the same as just setting the timezone in /etc/localtime instead, but using this has downsides:`

Reply:
I think this is a good compromise that documents the proper way without dissuading the option's use.

Reply:
Do you think we should list the downsides themselves or just say they exist?

Reply:
I'd rather not lose them, but this file itself might be the wrong place. Perhaps the right place is the rc.conf documentation on the docs page?

Reply:
Do you think we should revert this one https://github.com/void-linux/void-docs/commit/dc353ddeaf826c1eb8642dd92d52114389b7319b and then add some more stuff to it?

Reply:
I do think so.

Reply:
What is the point of "without dissuading the option's use", why would there be a reason to do that? What would be the difference to the hostname option? Why document a bad way of configuring this, instead of just the proper way?

Reply:
Unless the purpose is to depreciate this option, then it should be documented as non-ideal.

Reply:
@Vaelatern deprecating the option doesn't mean it's removed, right? It's not recommended, but will continue working for everyone who does use it. For new users, we make it clear what the best way of proceeding is, but the option to set it is still there.

Reply:
To depreciate an option is to set it up for later removal after everybody has migrated away.